docs/Examples/Response.md

```diff
@@ -104,7 +104,7 @@ provider.request(.zen) { result in
 If you have specific needs there are also the more general:
 
 - `filter(statusCode: Int)` which only accepts a single status code and throws an error otherwise.
-- `filter(statusCodes: ClosedRange<Int>)` which accepts a range of status codes and throws an error if the response's status code doesn't fall within the range.
+- `filter<R: RangeExpression>(statusCodes: R) throws -> Response where R.Bound == Int` which accepts any `RangeExpression` where the range's bound is an `Int` and throws an error if the response's status code doesn't fall within the range.
```

Comment:
How about removing the part about the return type? Like in the above `filter(statusCode: Int)`?